docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md

```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
+#### Symbol Header File
 
-### Read-Only Symbol Directives
+By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). The **Symbol Header File:** section in the **Resource Includes** dialog in Visual C++, lets you change the name of this header file. Enter a new file name in the section's edit box.
+
+> [!NOTE]
+> resource files not located in the same directory as the .RC file must prepend a relative path with escaped-'\\' to be read properly.
 
-Although Visual C++ only edits one header file for any given .RC file, Visual C++ supports references to symbols defined in additional read-only header files. Using the **Resource Includes** command from the **View** menu in Visual C++, you can specify any number of additional read-only header files as Read-Only Symbol Directives. The "read-only" restriction means that when you add a new resource in the .RC file, you can use a symbol defined in the read-only header file; but if you delete the resource, the symbol still remains defined in the read-only header file. You cannot change the numeric value assigned to a read-only symbol.
+#### Read-Only Symbol Directives
 
-### Compile-Time Directives
+Although Visual C++ only edits one header file for any given .RC file, Visual C++ supports references to symbols defined in additional read-only header files. The **Read-Only symbol Directives:** section in the **Resource Includes** dialog in Visual C++, lets you specify any number of additional read-only header files as Read-Only Symbol Directives. The "read-only" restriction means that when you add a new resource in the .RC file, you can use a symbol defined in the read-only header file; but if you delete the resource, the symbol still remains defined in the read-only header file. You cannot change the numeric value assigned to a read-only symbol.
+
+#### Compile-Time Directives
 
-Visual C++ also supports nesting of resource files, where one .RC file is #include'd within another. When you edit a given .RC file using Visual C++, any resources in the #include'd files are not visible. But when you compile the .RC file, the #include'd files are also compiled. Using the **Resource Includes** command from the **View** menu in Visual C++, you can specify any number of #include'd .RC files as Compile-Time Directives.
+Visual C++ also supports nesting of resource files, where one .RC file is #include'd within another. When you edit a given .RC file using Visual C++, any resources in the #include'd files are not visible. But when you compile the .RC file, the #include'd files are also compiled. The **Compile-Time Directives:** section in the **Resources Includes** dialog in Visual C++, lets you specify any number of #include'd .RC files as Compile-Time Directives.
```

Comment:
Visual C++ also supports nesting of resource files, where one .RC file is `#include`'d within another. When you edit a given .RC file using Visual C++, any resources in the `#include`'d files are not visible. But when you compile the .RC file, the `#include`'d files are also compiled. The **Compile-Time Directives:** section in the **Resources Includes** dialog in Visual C++ lets you specify any number of `#include`'d .RC files as Compile-Time Directives.